Recipe_2.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Recipe_2</title>
+</head>
+<body>
+  <header>
+    <nav>
+     <a href="index.html" class="logo">Happy<span1>Kitchen</span></a>
+    </nav>
+    <ul class="navbar">
+      <li><a href="Recipe_1.html" class="nav-list">Pi<span1>zza</span1> |</a></li>
+      <li><a href="Recipe_2.html" class="nav-list"><span1>Bur</span1>ger |</a></li>
+      <li><a href="Recipe_3.html" class="nav-list">Sa<span1>lad</span1> |</a></li>
+    </ul>
+  </header>
+  <img src="/images/burger-image.jpg" alt="burger image" class="bugger-image">
+  <div class="burger-ingredients">
+    <h4 class="classic-burger"><span>Classic</span> <span1>Burger</span1> <span>Ingredients</span></h4>
+    <nav class="list-all-ingredients">
+      <ul class="burger-ingredient-list">
+        <li>1 pound ground lean (7% fat) beef</li>
+        <li>1 large egg</li>
+        <li>½ cup minced onion</li>
+        <li>¼ cup fine dried bread crumbs</li>
+        <li>1 tablespoon Worcestershire</li>
+        <li>1 or 2 cloves garlic, peeled and minced</li>
+        <li>About 1/2 teaspoon salt</li>
+        <li>About 1/4 teaspoon pepper</li>
+        <li>4 hamburger buns (4 in. wide), split</li>
+        <li>About 1/4 cup mayonnaise</li>
+        <li>About 1/4 cup ketchup</li>
+        <li>4 iceberg lettuce leaves, rinsed and crisped</li>
+        <li>1 firm-ripe tomato, cored and thinly sliced</li>
+        <li>4 thin slices red onion</li>
+      </ul>
```

Comment:
Apply as required above.

Also, look at the display below, while you have done a great job hear, the div appears no to be well centered. Please fix that.

![screenshot-127 0 0 1_5500-2022 11 17-12_05_32](https://user-images.githubusercontent.com/99715640/202438757-653bf9ee-fff0-4498-b215-9365ad477496.png)